utils/estimateDynamicsUsingIntervals.m

```diff
@@ -36,20 +36,22 @@
          end
          newOrdering=true;
          
-         % extract sub-intervals in the case of walking datasets
-         isLeftLegIntervDef = ismember('leftLeg',intervalsNames);
-         isRightLegIntervDef = ismember('rightLeg',intervalsNames);
-         if(isLeftLegIntervDef || isRightLegIntervDef)
-             [~, initStanceLeft, endStanceLeft, initStanceRight, endStanceRight] = getStancePeriodsFromWalkingFTdata(...
-                 dataset.time, dataset.ftData.left_foot,...
-                 dataset.time, dataset.ftData.right_foot,...
-                 input.gait.minSwingLength, input.gait.tolerancePercentage, input.gait.shrinkPercentage);
-             % set the sub-intervals
-             if(isLeftLegIntervDef)
-                 input.intervals.leftLeg=struct('initTime',initStanceLeft,'endTime',endStanceLeft,'contactFrame','l_sole');
-             end
-             if(isRightLegIntervDef)
-                 input.intervals.rightLeg=struct('initTime',initStanceRight,'endTime',endStanceRight,'contactFrame','r_sole');
+         if (strcmp('walking',input.type))
```

Comment:
you mean, for having a default value in case the field is not defined in the `params.m` file?

Reply:
i mean something like this 

if (any(strcmp('type', fieldnames(input))))

because otherwise you can not run previously collected datasets since they would be lacking that and it would give you an error so instead of changing each params.m file just add the check

Reply:
Nice, I see you did the same for other options. Will push a new commit with the fix.